app/views/pregnancies/_patient_information.html.erb

```diff
@@ -77,8 +77,28 @@
                        options_for_select(referred_by_options,
                                           pregnancy.referred_by),
                                           autocomplete: 'off' %>
-          <%= f.text_field :special_circumstances, autocomplete: 'Off' %>
-        </div>
+         
+          
+          <%= f.label :special_circumstances, 'Special circumstances' %> 
```

Comment:
Special circumstances won't have any heading. I tried to put the label in the form group line and the styling went wonky.